src/test/java/SeleniumTestngTest.java

```diff
@@ -1,24 +1,121 @@
+        }
+    }
+    @Test
+    public void testDropdown() throws InterruptedException {
+        browser.get("https://the-internet.herokuapp.com/dropdown");
+        WebElement dropdown = browser.findElement(By.cssSelector("#dropdown"));
+        Thread.sleep(1000);
+        Select select = new Select(dropdown);
+        Thread.sleep(1000);
+        select.selectByVisibleText("Option 2");
+        Thread.sleep(1000);
+        Assert.assertEquals(select.getFirstSelectedOption().getText(), "Option 2", "'Option 2' is not selected.");
+    }
+    @Test
+    public void testFormAuthentication() throws InterruptedException {
+        browser.get("https://the-internet.herokuapp.com/login");
+        WebElement username = browser.findElement(By.cssSelector("#username"));
+        WebElement password = browser.findElement(By.cssSelector("#password"));
+        WebElement loginButton = browser.findElement(By.cssSelector("#login > button > i"));
+        username.sendKeys("tomsmith");
+        Thread.sleep(1000);
+        password.sendKeys("SuperSecretPassword!");
+        Thread.sleep(1000);
+        loginButton.click();
+        WebElement successMessage = browser.findElement(By.cssSelector("#flash"));
+        Assert.assertTrue(successMessage.getText().contains("You logged into a secure area!"));
+        Thread.sleep(1000);
+        WebElement logoutButton = browser.findElement(By.cssSelector("#content > div > a"));
+        logoutButton.click();
+        Thread.sleep(1000);
+        WebElement logoutMessage = browser.findElement(By.cssSelector("#flash"));
+        Assert.assertTrue(logoutMessage.getText().contains("You logged out of the secure area!"));
+    }
+    @Test
+    public void testDragAndDrop() throws InterruptedException {
+        browser.get("https://the-internet.herokuapp.com/drag_and_drop");
+        WebElement columnA = browser.findElement(By.xpath("//*[@id=\"column-a\"]"));
+        WebElement columnB = browser.findElement(By.xpath("//*[@id=\"column-b\"]"));
+        Actions actions = new Actions(browser);
+        actions.dragAndDrop(columnA, columnB).perform();
+        String columnAText = columnA.getText();
+        Thread.sleep(2000);
+        String columnBText = columnB.getText();
+        Assert.assertEquals(columnAText, "B");
+        Assert.assertEquals(columnBText, "A");
+    }
+    @Test
+    public void testHorizontalSlider() throws InterruptedException {
+        browser.get("https://the-internet.herokuapp.com/horizontal_slider");
+        WebElement slider = browser.findElement(By.cssSelector("#content > div > div > input[type=range]"));
+        Actions moveSlider = new Actions(browser);
+        moveSlider.clickAndHold(slider).moveByOffset(30, 0).release().perform();
+        Thread.sleep(2000);
+        WebElement sliderValue = browser.findElement(By.cssSelector("#range"));
+        Assert.assertNotEquals(sliderValue.getText(), "0");
```

Comment:
There can be a better way to manipulate the slider so you would be able to not just check that the current value is "not 0", but look for a precise value

You'll need to find a way to move the slider for ex. by 2 positions and make sure the current selected value is `2`.
Moving by offset may depend on the current screen resolution, but there's a way to make it screen-resolution-agnostic ;)